File: Data Structures & Algorithms/two-integer-sum/submission-11.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indices = {}
        for i, num in enumerate(nums):
            indices[num] = i
        for i in range(len(nums)):
            diff = target - nums[i]
            if diff in indices and indices[diff] != i:
                return sorted([i, indices[diff]])

File: Data Structures & Algorithms/is-anagram/submission-11.go
func isAnagram(s string, t string) bool {
    if len(s) != len(t) {
        return false
    }

    var count [26]int

    for i:=0;i<len(s);i++ {
        count[s[i]-'a']++
        count[t[i]-'a']--
    }

    for _, num := range count {
        if num != 0 {
            return false
        }
    }
    return true
}
